### ISSUE TITLE
Release 0.1.7

### ISSUE THREAD

**Issue by blimmer** [samselikoff/ember-cli-mirage#243]
We should do a release now that the new Pretender allowing host support is live.

**Comment by kiwiupover** [comment]
yes please :+1:

**Comment by seawatts** [comment]
:+1:

**Comment by samselikoff** [comment]
I can cut one, is there anything special needed? I actually haven't had time to try this myself. I'd like to add it to the docs. Is it just

```js
this.get('http://www.youtube.com', ...
```

?

**Comment by blimmer** [comment]
I actually couldn't get that working (defining the cors host in your `this.get`). 

As far as I can tell, you have to set it as your namespace, e.g.

```js
  this.namespace = 'http://api.example.org';
  this.get('foos');
```

**Comment by blimmer** [comment]
I think we need to fix #244 before we do this release.
